lib/launcher.js

```diff
@@ -351,11 +351,16 @@ class Launcher {
 
     /**
      * emit event from child process to reporter
-     * @param  {Object} m  event object
+     * @param  {String} cid
+     * @param  {Object} m event object
      */
-    messageHandler (m) {
+    messageHandler (cid, m) {
```

Comment:
I think this is a little confusion in the names of variables. But the consequences of this confusion will be difficult to correct. Because in reporters everywhere is used `cid` (which is actually `rid`)...
I'm not mistaken?